x/subscription/keeper/subscription.go

```diff
@@ -205,7 +205,12 @@ func (k Keeper) advanceMonth(ctx sdk.Context, subkey []byte) {
 	consumer := string(subkey)
 
 	var sub types.Subscription
-	_ = k.subsFS.FindEntry(ctx, consumer, block, &sub)
+	if found := k.subsFS.FindEntry(ctx, consumer, block, &sub); !found {
+		// subscription (monthly) timer has expired for an unknown subscription:
+		// either the timer was set wrongly, or the subscription was incorrectly
+		// removed; and we cannot even return an error about it.
+		panic("subscription month expiry with unknown key " + consumer)
```

Comment:
lets not panic, just bail with an error print

Reply:
fixed: e88390d